keel-sql/src/test/kotlin/com/netflix/spinnaker/keel/sql/TestDatabase.kt

```diff
@@ -64,23 +59,25 @@ internal fun initDatabase(jdbcUrl: String): DSLContext {
 
 internal class DatabaseInitializationFailed(cause: Throwable) : RuntimeException(cause)
 
-internal fun DSLContext.flushAll(schema: String = "PUBLIC") =
-  meta()
-    .schemas
-    .filter { it.name == schema }
-    .flatMap(Schema::getTables)
-    .forEach {
-      truncate(it).execute()
+internal fun DSLContext.flushAll(schema: String = "keel") {
+  LiquibaseConfiguration
+    .getInstance()
+    .getConfiguration(GlobalConfiguration::class.java)
+    .apply {
+      meta()
+        .schemas
+        .filter { it.name == schema }
+        .flatMap(Schema::getTables)
+        .filterNot {
+          it.name in setOf(
+            databaseChangeLogTableName,
+            databaseChangeLogLockTableName
+          )
```

Comment:
D'oh! If I truncate these tables the following test will fall over trying to re-run the Liquibase changeset.